### DISCUSSION TITLE
The harvester and the farmer can be connected but not run.

### DISCUSSION THREAD

**Discussion by guojiazhuxi** [Chia-Network/chia-blockchain#5989] [Support]
The farmer's journal (about the harvester)
farmer farmer_server              : DEBUG    -> handshake to peer 192.168.xxxx
farmer farmer_server              : DEBUG    -> harvester_handshake to peer 192.168.xxxxx
the harvester's journal
 harvester chia.plotting.plot_tools: INFO     Loaded a total of 86 plots of size 8.827717046642647 TiB, in 5.460477352142334 seconds
 harvester harvester_server        : DEBUG    Time taken to process harvester_handshake from xxxxxx is sec

There is no journal about them later.But in the example, there are other logs after the log, such as：
[time stamp] farmer farmer_server   : INFO   -> new_signage_point_harvester to peer [harvester IP address] [peer id - 64 char hexadecimal]
[time stamp] farmer farmer_server   : INFO   <- farming_info from peer [peer id - 64 char hexadecimal] [harvester IP address]
[time stamp] farmer farmer_server   : INFO   <- new_proof_of_space from peer [peer id - 64 char hexadecimal] [harvester IP address]

Why is that?

**Comment by guojiazhuxi** [comment]
It's very difficult to solve, but if I use the window system, it can be used normally. I don't know why

**Reply by terryxsq** [reply]
hey, i just found out that those INFO are now DEBUG. So if you set those to DEBUG log level then you will see them.

你需要设置成DEBUG的才能显示那些示例中的log。

**Comment by BrandtH22** [comment]
@guojiazhuxi , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
